### DISCUSSION TITLE
errors version 6

### DISCUSSION THREAD

**Discussion by sluc69** [Kunena/Kunena-Forum#8348] [General]
[kunena 6 errors.txt](https://github.com/Kunena/Kunena-Forum/files/7038158/kunena.6.errors.txt)

Installed version 6 on  ubuntu 20.4 with latest apache2 .
Copied records from the old forum (version 5.x on J3) to new DB tables.
The categories and topics are displayed correctly so far but other errors and display errors occur

**Comment by xillibit** [comment]
Hello,

Thanks for the report, did-you have used the Kuenna 6.0 beta2 ? Which Php version are-you using ?

**Reply by sluc69** [reply]
I installed beta1 then beta 2 now updates to RC1
PHP 7.4.x (latest)
MySQL: 5.7.35-0ubuntu0.20.4

**Reply by xillibit** [reply]
We haven't released yet a RC1 of Kunena 6.0, it was done by mistake but it was a beta1. Things related to RC1 has been removed

If you want a most recent version of Kunena 6.0 you need to install the nightly build available here in he bottom of the page : https://www.kunena.org/download

**Comment by iki007** [comment]
Version 6. Beta7, updated from Kunena  5.2.8 on joomla 3.10 to  joomla 4.0.6
 PHP 7.4.27
 MySQLi 10.3.27-MariaDB

When i want to see the messages from topics, i receive this error: 
_kunena_private' doesn't exist
![Screenshot from 2022-02-12 15-52-57](https://user-images.githubusercontent.com/6887209/153714141-71b1dc59-9fce-464a-8584-b2838d70efef.png)

**Reply by 810** [reply]
i have created a new nightly build on https://www.kunena.org/download please try that build. But for the database update you need the k5.2  data again and structure.

**Reply by sluc69** [reply]
Currently upgraded the test setup to 6.0.0-RC2-GIT/2022-04-07. It looks fairly stable.
Is there an upgrade route/plan/guidelines from 5.x to 6.0??
Is the database kunena-tables structure still the same in 6.0 as in 5.x?
